metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

```diff
@@ -7,14 +7,58 @@ See the
 - Enhance admin APIs responses with detailed metadata
 ## Concept mapping 
 
-| Power BI              | Datahub             |                                                                                               
+| Power BI              | Datahub                 |                                                                                               
 |-----------------------|---------------------|
 | `Dashboard`           | `Dashboard`         |
-| `Dataset, Datasource` | `Dataset`           |
+| `Dataset's Table`     | `Dataset`           |
 | `Tile`                | `Chart`             |
 | `Report.webUrl`       | `Chart.externalUrl` |
 | `Workspace`           | `N/A`               |
 | `Report`              | `Dashboard`         |
 | `Page`                | `Chart`             |
 
 If Tile is created from report then Chart.externalUrl is set to Report.webUrl.
+
+## Lineage
```

Comment:
Add one sentence here to descibe what type of lineage this source extracts

Reply:
(Introduction sentence)

Reply:
Done